aviary/variable_info/enums.py

```diff
@@ -64,7 +64,7 @@ class EquationsOfMotion(Enum):
 
 
 @unique
-class GASPEngineType(Enum):
```

Comment:
per your comment in the PR description, is it possible to add custom setters that allow us to set them by string?

Reply:
Yeah, I should be able to create a setter that allows you to set with an int, a string, or the specific enum object.